grails-app/controllers/mama/ng/scheduler/MessageController.groovy

```diff
@@ -0,0 +1,110 @@
+package mama.ng.scheduler
+
+import grails.converters.JSON
+
+class MessageController {
+
+    static allowedMethods = [save: "POST", update: "PUT", delete: "DELETE"]
+
+    def cronParserService
+
+    def list() {
+        params.max = Math.min(params.max ? params.int('max') : 50, 200)
+        def list = Message.list(params)
+        withFormat {
+            json {
+                response.status = 200
+                render list as JSON
+            }
+        }
+    }
+
+    def show(Message instance) {
+        withFormat {
+            json {
+                response.status = 200
+                render instance as JSON
+            }
+        }
+    }
+
+    def create(Message instance) {
+        if (!instance.nextSend) {
+            instance.nextSend = cronParserService.determineNextDate(instance.schedule.cronDefinition)
```

Comment:
Oh wait, this is for creating a message, not a schedule. Does it make any sense to create a message with nextSend null?